test/Interop/Cxx/class/Inputs/loadable-types.h

```diff
@@ -155,4 +155,11 @@ struct StructDeletedDestructor {
   ~StructDeletedDestructor() = delete;
 };
 
+struct DummyStruct {};
+
+struct HasUserProvidedDestructorAndDummy {
+  DummyStruct dummy;
```

Comment:
What's the significance of the data member? Would be nice to provide some explanation in the tests that use this new type.